up42/__init__.py

```diff
@@ -41,11 +41,11 @@ def initialize_project() -> "Project":
     return Project(auth=_auth, project_id=_auth.project_id)
 
 
-def initialize_catalog(backend: str = "ONE_ATLAS") -> "Catalog":
+def initialize_catalog() -> "Catalog":
```

Comment:
So the ONEATLAS thing is not required anymore by catalog search?

Reply:
As far as I can see it is not required, will confirm with backend on Monday. Examples with and without it in postman, but does not seem to make a difference, without & without it same: possible combinations of sensors, number of images, effect of limit parameter.